_about/computation.md

```diff
@@ -8,20 +8,20 @@ Jobs run on the OSG will be able to execute on servers at numerous remote physic
 
 1. Independent compute tasks using up to 8 cores (ideally 1 core, each), less than 8 GB memory (RAM) per core, and 1 GPU, and running for 1-12 hours. *Additional capabilities for COVID-19 research are currently available, with up to 48 hours of runtime per job.* Please contact the support listed below for more information about these capabilities. Application-level checkpointing can be implemented for longer-running work (for example, applications writing out state and restart files). Workloads with independent jobs of 1 core and less than 1 GB RAM are ideal, with up to thousands of concurrently-running jobs and 100,000s of hours achieved daily. Jobs using several cores and/or several GB of RAM will likely experience hundreds of concurrently-running jobs.
 2. Compute sites in the OSG can be configured to use pre-emption, which means jobs can be automatically killed if higher priority jobs enter the system. Pre-empted jobs will restart on another site, but it is important that the jobs can handle multiple restarts and/or complete in less than 12 hours.
-3. Software dependencies can be can be staged with the job, distributed via containers, or installed on the read-only distributed OASIS filesystem (which can also support software modules). Statically-linked binaries are ideal. However, dynamically linked binaries with standard library dependencies, built for 64-bit Red Hat Enterprise Linux (RHEL) version 6 or 7 will also work. OSG can support some licensed software (like Matlab, Matlab-Simulink, etc.) where compilation allows execution without a license, or where licenses still accommodate multiple jobs and are not node-locked.
+3. Software dependencies can be can be staged with the job, distributed via containers, or installed on the read-only distributed OASIS filesystem (which also supports OSG-curated software modules). Statically-linked binaries are ideal. However, dynamically linked binaries with standard library dependencies, built for 64-bit Red Hat Enterprise Linux (RHEL) version 6 or 7 will also work. OSG can support some licensed software (like Matlab, Matlab-Simulink, etc.) where compilation allows execution without a license, or where licenses minimally restrict the number of running jobs.
```

Comment:
The wording here is awkward.  Sure dynamically linked binaries are fine for more then RHEL 6 or 7, dynamically linked binaries are fine for ubuntu as well, with the right container.  Should you include anything about dynamically linked things at all, if containers solve that problem mostly.

Reply:
How would you put it?